src/Engine/Manager/ResourceManager.hpp

```diff
@@ -24,30 +24,9 @@ class ResourceManager {
     friend class Hub;
     
     public:
-        /// Create a rectangle for rendering if it doesn't already exist.
-        /**
-         * @return The rectangle instance
-         */
-        Video::Geometry::Rectangle* CreateRectangle();
-        
-        /// Free the reference to the rectangle.
-        /**
-         * Deletes the instance if no more references exist.
-         */
-        void FreeRectangle();
-        
-        /// Create a cube for rendering if it doesn't already exist.
-        /**
-         * @return The cube instance
-         */
-        Geometry::Cube* CreateCube();
-        
-        /// Free the reference to the cube.
-        /**
-         * Deletes the instance if no more references exist.
-         */
-        void FreeCube();
-        
+        /// Constructor
+        ResourceManager() {}
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'ResourceManager::rectangle' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'ResourceManager::cube' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)